### PR TITLE
#164694231 List user functionality

### PR DESCRIPTION
**Pull request by Rhytah**
#### What does this Pr do?
* List  author profiles

#### Description of Task to be completed?
- [x]  add profile list renderer
- [x]  add profile list serializer class
- [x]  add the profile list view
- [x]  define endpoint route for  functionality
#### How should this be manually tested?
* Run application server using command `python manage.py runserver`
- Log in with a verified account
- access the `GET/api/profiles/` endpoint
- You should be able to see a list of author profiles

#### What are the relevant pivotal tracker stories?
[#164694231](https://www.pivotaltracker.com/story/show/164694231)

### CODE REVIEW

File: authors/apps/profiles/views.py [hunk 1]
```diff
@@ -44,3 +45,13 @@ def get_object(self):
         if_owner_permission(self.request, **self.kwargs)
         username = self.kwargs.get("username")
         return get_object_or_404(Profile, user__username=username)
+
+
+class UserProfileListView(generics.ListAPIView):
```
Comment (kimbugp) [inline authors/apps/profiles/views.py:50]: @Rhytah , add a docstring to you class

**Review by kimbugp — CHANGES_REQUESTED** [review]

**Comment by kimbugp** [comment]
@Rhytah , rebase your branch with develop

File: authors/apps/profiles/views.py (hunk 1 above)
Comment (Rhytah) [reply authors/apps/profiles/views.py:50]: @kimbugp ok, let me do that

**Review by kimbugp — APPROVED** [review]

**Review by ghost — COMMENTED** [review]
Good work, just fix the codacy compliants.

**Pull request merged**